packages/kbn-utils/src/path/index.ts

```diff
@@ -27,6 +27,8 @@ const CONFIG_DIRECTORIES = [
 
 const DATA_PATHS = [join(REPO_ROOT, 'data'), '/var/lib/kibana'].filter(isString);
 
+const LOGS_PATHS = [join(REPO_ROOT, 'logs'), '/var/lib/kibana'].filter(isString);
```

Comment:
__issue:__ shouldn't we use `/var/log/kibana` instead?
```suggestion
const LOGS_PATHS = [join(REPO_ROOT, 'logs'), '/var/log/kibana'].filter(isString);
```

Reply:
Goot catch 👍